deepchem/models/tests/test_layers.py

```diff
@@ -613,6 +613,6 @@ def test_scale_norm():
   input_ar = torch.tensor([[1., 99., 10000.], [0.003, 999.37, 23.]])
```

Comment:
Can you add unit tests for these new classes?

Reply:
Working on the tests now :+1: